manager/burrow-mint/pipe.go

```diff
@@ -409,6 +411,10 @@ func (pipe *burrowMintPipe) DumpStorage(address []byte) (*rpc_tm_types.ResultDum
 // TODO: [ben] resolve incompatibilities in byte representation for 0.12.0 release
 func (pipe *burrowMintPipe) Call(fromAddress, toAddress, data []byte) (*rpc_tm_types.ResultCall,
 	error) {
+	if vm.RegisteredNativeContractAddress(toAddress) {
+		return nil, errors.New("Native contracts can not be called directly." +
+			"Use a deployed contract that calls the s")
```

Comment:
unfinished error message, include `toAddress`, spacing between sentences and that native contracts cannot be called directly "... from RPC/pipe".

please apply `LeftPadWord256` here explicitly.

Even though this error is caught before entering execution, and additionally it would hit L420 `GetAccount` and should fail; but it is a good catch because we should consider blocking out the (creation and) existence of contracts at a registered native contracts)